src/core/load-fragment.c

```diff
@@ -5687,6 +5687,7 @@ int config_parse_cgroup_socket_bind(
                 return log_oom();
         *item = (CGroupSocketBindItem) {
                 .address_family = af,
+                .ip_protocol = IPPROTO_IP,
```

Comment:
IPPROTO_IP is such a weird concept, i.e. the IP protocol doesn't have an L4 proto IP. maybe add a comment `/* no protocol specified for now */`